x/incentives/keeper/distribute_test.go

```diff
@@ -369,19 +369,18 @@ func (s *KeeperTestSuite) TestDistribute_ExternalIncentives_NoLock() {
 
 	type test struct {
 		// setup
-		isPerpertual       bool
-		tokensToAddToGauge sdk.Coins
-		gaugeStartTime     time.Time
-		gaugeCoins         sdk.Coins
-		distrTo            lockuptypes.QueryCondition
-		startTime          time.Time
-		numEpochsPaidOver  uint64
-		poolId             uint64
+		isPerpertual      bool
+		gaugeStartTime    time.Time
+		gaugeCoins        sdk.Coins
+		distrTo           lockuptypes.QueryCondition
+		startTime         time.Time
+		numEpochsPaidOver uint64
+		poolId            uint64
 
 		// expected
 		expectErr                              bool
 		expectedDistributions                  sdk.Coins
-		expectedRemainingAmountIncentiveRecord []sdk.Dec
+		expectedRemainingAmountIncentiveRecord []sdkmath.LegacyDec
```

Comment:
nit: shall we use `osmomath.Dec` here?

Reply:
https://github.com/osmosis-labs/osmosis/pull/6288/files#diff-d5967ddef40ae47ef4140b9c47026cac85b85d9d04173b271c6c23dd182abdbcR408

https://github.com/osmosis-labs/osmosis/pull/6288/files#diff-d5967ddef40ae47ef4140b9c47026cac85b85d9d04173b271c6c23dd182abdbcR425

and etc

Reply:
It is a very good nit, and I wasn't sure. 

@p0mvn has been moving things over to use cosmossdk.io/math but I think this is a great question to bring up.  Thanks!

Reply:
(let's not merge this till there's an answer to this question)